io_u.c

```diff
@@ -1326,8 +1326,10 @@ static struct fio_file *__get_next_file(struct thread_data *td)
 	if (f && fio_file_open(f) && !fio_file_closing(f)) {
 		if (td->o.file_service_type == FIO_FSERVICE_SEQ)
 			goto out;
-		if (td->file_service_left--)
-			goto out;
+		if (td->file_service_left) {
+		  td->file_service_left--;
+		  goto out;
+		}
```

Comment:
Is this fixing a different bug? If it's a patch already submitted maybe you can rebase to check if it's already in place?

Reply:
It was bug I found during work on this. Has already been merged.